bin/create-bundle-alpine.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
This is the script that creates custom airgap bundles at runtime. The server adds it to a ConfigMap that the Job mounts. The server then passes the signed PUT url as the first arg and a list of URLs to individual packages to include.